internal/controller/operatorconfigmap_controller.go

```diff
@@ -694,6 +723,20 @@ func (c *OperatorConfigMapReconciler) reconcileClientOperatorSubscription() erro
 	return nil
 }
 
+func (c *OperatorConfigMapReconciler) reconcileNoobaaOperatorSubscription() error {
+	noobaaSubscription, err := c.getSubscriptionByPackageName("noobaa-operator")
```

Comment:
Not sure how this calculates the correct subscription when the client is not on the same cluster as the provider. Can you please provide an explanation?